File: web/src/app/types/castingcall.ts
export class CastingCall {

  id: number;
  projectName: string;
  projectDetails: string;
  productionCompany: string;
  roleDetails: string;
  address: string;
  startAge: number;
  endAge: number;
  gender: number;
  countryId: number;
  auditionVenue: number;
  startDate: string;
  endDate: string;
  time: string;
  roleIds: string[];

  constructor(res: any) {

   this.id = res['id'];
   this.projectName = res['projectName'];
   this.projectDetails = res['projectDetails'];
   this.productionCompany = res['productionCompany'];
   this.roleDetails = res['roleDetails'];
   this.address = res['address'];
   this.startAge = res['startAge'];
   this.endAge = res['endAge'];
   this.gender = res['gender'];
   this.countryId = res['countryId'];
   this.auditionVenue = res['cityId'];
   this.startDate = res['startDate'];
   this.endDate = res['endDate'];
   this.time = res['hours'];
   this.roleIds = res['roleIds'];

  }
 }

File: android/app/src/main/java/com/samayu/scaction/dto/CastingCall.java
package com.samayu.scaction.dto;

import java.sql.Date;
import java.util.List;

/**
 * Created by NandhiniGovindasamy on 9/17/18.
 */

public class CastingCall {


    private long id,userId;


    private String projectName,projectDetails,productionCompany, roleDetails,address,hours,roleIds;


    private int startAge,endAge,gender,cityId,countryId;

    private Date startDate,endDate;

    private List<CastingCallApplication> userApplications;

    public long getId() {
        return id;
    }

    public void setId(long id) {
        this.id = id;
    }

    public long getUserId() {
        return userId;
    }

    public void setUserId(long userId) {
        this.userId = userId;
    }

    public String getProjectName() {
        return projectName;
    }

    public void setProjectName(String projectName) {
        this.projectName = projectName;
    }

    public String getProjectDetails() {
        return projectDetails;
    }

    public void setProjectDetails(String projectDetails) {
        this.projectDetails = projectDetails;
    }

    public String getProductionCompany() {
        return productionCompany;
    }

    public void setProductionCompany(String productionCompany) {
        this.productionCompany = productionCompany;
    }

    public String getRoleDetails() {
        return roleDetails;
    }

    public void setRoleDetails(String roleDetails) {
        this.roleDetails = roleDetails;
    }

    public String getAddress() {
        return address;
    }

    public void setAddress(String address) {
        this.address = address;
    }

    public String getHours() {
        return hours;
    }

    public void setHours(String hours) {
        this.hours = hours;
    }

    public String getRoleIds() {
        return roleIds;
    }

    public void setRoleIds(String roleIds) {
        this.roleIds = roleIds;
    }

    public int getStartAge() {
        return startAge;
    }

    public void setStartAge(int startAge) {
        this.startAge = startAge;
    }

    public int getEndAge() {
        return endAge;
    }

    public void setEndAge(int endAge) {
        this.endAge = endAge;
    }

    public int getGender() {
        return gender;
    }

    public void setGender(int gender) {
        this.gender = gender;
    }

    public int getCityId() {
        return cityId;
    }

    public List<CastingCallApplication> getUserApplications() {
        return userApplications;
    }

    public void setUserApplications(List<CastingCallApplication> userApplications) {
        this.userApplications = userApplications;
    }

    public void setCityId(int cityId) {
        this.cityId = cityId;
    }

    public int getCountryId() {
        return countryId;
    }

    public void setCountryId(int countryId) {
        this.countryId = countryId;
    }

    public Date getStartDate() {
        return startDate;
    }

    public void setStartDate(Date startDate) {
        this.startDate = startDate;
    }

    public Date getEndDate() {
        return endDate;
    }

    public void setEndDate(Date endDate) {
        this.endDate = endDate;
    }
}
